### ISSUE TITLE
Map-athon by PDRF to be hosted by HSBC

### ISSUE THREAD

**Issue by govvin** [OSMPH/events#2]
Currently discussing possible map-athon event next month with PDRF+HSBC over email. Will invite coordinators to this board after we've firmed up on event objectives.

**Current agenda:**
- [ ] outstanding report from first engagement
- [x] initial proposal
- [ ] event objectives
- [ ] date, venue
- [ ] staffing
- [ ] training/orientation of new batch of mappers from HSBC and Pitogo(?)
- [ ] mapping MSMEs in Pitogo using FieldPapers and attributes from internal database
- [ ] identify feasible projects for remote mapping by beginners (participants from first HSBC workshop). Recommend adoption of: http://tasks.hotosm.org/project/1458

**Comment by govvin** [comment]
Prepare program proposal

**Comment by seav** [comment]
Did PDRF/HSBC approach us again?

**Comment by govvin** [comment]
Not exactly, @seav. I had a chance encounter with someone from PDRF, and they mentioned HSBC is inquiring about what's next for them after the first engagement. I pitched for a map-athon, and they're keen on helping organize one with/for HSBC. 

I mentioned the chance meeting [here](https://osmph.slack.com/archives/0_o-core/p1455691204000053).
